index.js

```diff
@@ -5,63 +5,163 @@ const port = 3001
 const USERS = [];
 
 const QUESTIONS = [{
+    id: "1",
     title: "Two states",
     description: "Given an array , return the maximum of the array?",
     testCases: [{
         input: "[1,2,3,4,5]",
-        output: "5"
-    }]
-}];
-
-
-const SUBMISSION = [
-
-]
+        output: "5",
+      }, 
+    ],
+},
+{
+  id: "2",
+  title: "SQUATS",
+  description: "Somu went to the gym today. He decided to do X sets of squats. Each set consists of 15 squats. Determine the total number of squats that he did today.",
+  testCases: [{
+    input: "2",
+    output: "30",
+  },
+  {
+    input: "5",
+    output: "75",
+  },
+  {
+    input: "99",
+    output: "1485",
+  },
+  {
+    input: "1",
+    output: "15",
+  },
+],
+
+},
+{
+  id: "3",
+  title: "Tour of King",
+  description: "King loves to go on tours with his friends.King has N cars that can seat 5 people each and M cars that can seat 7 people each. Determine the maximum number of people that can travel together in these cars.",
+  testCases: [{
+    input: "{4,8}",
+    output: "76",
+  },
+  {
+    input: "{ 2, 13}",
+    output: "101",
+  },
+
+],
+
+},
+{
+  id: "4",
+  title: "Ageing",
+  description: "Chef's current age is 20 years, while Chefina's current age is 10 years. Determine Chefina's age when Chef will be X years old.",
+  testCases: [{
+    input: "25",
+    output: "15",
+  },
+  {
+    input: "36",
+    output: "26",
+  },
+  {
+    input: "54",
+    output: "44",
+  },
+
+],
+},
+
+
+];
+
+
+const SUBMISSION = [];
 
 app.post('/signup', function(req, res) {
   // Add logic to decode body
   // body should have email and password
+  const {email, password} = req.body;
 
 
   //Store email and password (as is for now) in the USERS array above (only if the user with the given email doesnt exist)
-
+  let userExists = false;
+  for(let user of USERS){
+    if(user.email === email){
+      userExists = true;
+    }
+    
+    
+  }
+  if(!userExists){
+    USERS.push({email, password});
```

Comment:
`Optional 💡` when dealing with passwords standard practice is to hash the password and store in your data store.

You can checkout [bcrypt](https://www.npmjs.com/package/bcrypt) if interested to implement with password hashing